### PR TITLE
Drone: specify branches to trigger pipelines

### PR DESCRIPTION
**Pull request by bk201**
Only trigger pipelines on these branches:
- master
- v*

This avoids running publish pipeline on mergify backport PR branches (e.g. `mergify/bp/v1.1/pr-385` of PR https://github.com/harvester/harvester-installer/pull/391. [Error](https://drone-publish.rancher.io/harvester/harvester-installer/1074))

### CODE REVIEW

**Review by guangbochen — APPROVED** [review]
LGTM, and wondering if the backport branch gets deleted automatically after PR merge? thanks.

**Comment by bk201** [comment]
> LGTM, and wondering if the backport branch gets deleted automatically after PR merge? thanks.

sure thing: please check https://docs.mergify.com/actions/backport/#backport
> Once the backporting pull request is closed or merged, Mergify will automatically delete the backport head branch that it created.

**Pull request merged**
